### PR TITLE
Cleaning the code a little and updating the poms.

### PR DESCRIPTION
**Pull request by ehsavoie**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -99,21 +101,4 @@
         <module>generic-jms-ra-jar</module>
         <module>generic-jms-ra-rar</module>
     </modules>
-
-    <repositories>
```
Comment (jmesnil) [inline pom.xml]: don't you need to keep to be able to release new versions?

File: generic-jms-ra-jar/pom.xml [hunk 2]
```diff
@@ -50,8 +50,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <version>3.1</version>
                 <configuration>
-                    <source>1.6</source>
```
Comment (jmesnil) [inline generic-jms-ra-jar/pom.xml]: changing the JDK version form 1.6 to 1.8 seems important enough to be mentioned in the commit message

File: pom.xml (hunk 1 above)
Comment (ehsavoie) [reply pom.xml]: This is the configuration for where to pull artifacts from, distributionManagement is the part on where to deploy artifacts to.

File: generic-jms-ra-jar/pom.xml (hunk 2 above)
Comment (ehsavoie) [reply generic-jms-ra-jar/pom.xml]: I agree

**Pull request merged**